### ISSUE TITLE
Chalice Deploy doesn't work & returns useless message

### ISSUE THREAD

**Issue by adamleo** [aws/chalice#704]
Hi guys, when I was using `chalice deploy` on a sample app was fine, now I am trying to use the **request** library suddenly it doesn't work anymore.

Can anybody give some advice?

Below is my code:

```
from chalice import Chalice
from botocore.vendored import requests


app = Chalice(app_name='hitter')


@app.route('/')
def index():
	return {'lambda': 'hitter'}


@app.route('/hit/city/{city}')
def index(city):
	return requests.get("https://fcyax2nf6g.execute-api.us-east-1.amazonaws.com/api/cities/" + city).text
```

And here's the response I got:
```
(chalice) my_mac:hitter hongru.liu$ chalice deploy
Creating role: hitter-dev
'requests'
```

The message was just a string says `'requests'`, I have no idea what is going on.

Thanks a lot!

**Comment by adamleo** [comment]
Sorry my bad, just realised that it was caused by the duplicated function name.

**Comment by sbassi** [comment]
I think that the "useless message" part is still standing.
We could put an enhancement proposal to give an error telling the root cause of the problem instead of "request"

**Comment by skwashd** [comment]
The "useless message" problem was fixed in #727.
